packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts

```diff
@@ -27,17 +27,36 @@ describe('purify', () => {
 
   it('prefixes safe default imports with /*@__PURE__*/', () => {
     const input = tags.stripIndent`
-      /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = __webpack_require__("rlar");
-      /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__);
+    /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = __webpack_require__("rlar");
+    /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__);
       /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0_zone_js___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_0_zone_js__);
       /** PURE_IMPORTS_START rxjs_Subject,zone_js PURE_IMPORTS_END */
-    `;
+      `;
     const output = tags.stripIndent`
       /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = /*@__PURE__*/__webpack_require__("rlar");
       /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject___default = /*@__PURE__*/__webpack_require__.n(__WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__);
       /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0_zone_js___default = /*@__PURE__*/__webpack_require__.n(__WEBPACK_IMPORTED_MODULE_0_zone_js__);
       /** PURE_IMPORTS_START rxjs_Subject,zone_js PURE_IMPORTS_END */
-    `;
+      `;
+
+    expect(tags.oneLine`${purify(input)}`).toEqual(tags.oneLine`${output}`);
+  });
+
+  it('finds old matches that started with dots', () => {
+    const input = tags.stripIndent`
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__util_root__ = __webpack_require__(13);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_1__util_toSubscriber__ = __webpack_require__(67);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_2__symbol_observable__ = __webpack_require__(45);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_3__util_pipe__ = __webpack_require__(71);
+        /** PURE_IMPORTS_START ._util_root,._util_toSubscriber,.._symbol_observable,._util_pipe PURE_IMPORTS_END */
+      `;
+    const output = tags.stripIndent`
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__util_root__ = /*@__PURE__*/__webpack_require__(13);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_1__util_toSubscriber__ = /*@__PURE__*/__webpack_require__(67);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_2__symbol_observable__ = /*@__PURE__*/__webpack_require__(45);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_3__util_pipe__ = /*@__PURE__*/__webpack_require__(71);
+        /** PURE_IMPORTS_START ._util_root,._util_toSubscriber,.._symbol_observable,._util_pipe PURE_IMPORTS_END */
```

Comment:
where do the dots come from? can you leave a comment in the code explaining that?

Reply:
Fixed